train_configs/llama3_8b.toml

```diff
@@ -14,6 +14,8 @@ profile_freq = 100
 log_freq = 10
 enable_tensorboard = true
 save_tb_folder = "tb"
+enable_wandb = true
+wandb_config = { project = "torchtitan", entity = "your-team" }
```

Comment:
to increase visibility, let's move this to `debug_model.toml` (and default to false)